app/app/services/aggregators/response_objects/identity.rb

```diff
@@ -27,7 +29,8 @@ def self.from_argyle(identity_response_body)
         date_of_birth: identity_response_body["birth_date"],
         emails: [ identity_response_body["email"] ],
         ssn: Aggregators::FormatMethods::Argyle.obfuscate_ssn(identity_response_body["ssn"]),
-        phone_numbers: [ identity_response_body["phone_number"] ]
+        phone_numbers: [ identity_response_body["phone_number"] ],
+        zip_code: identity_response_body["address"]["postal_code"]
```

Comment:
```suggestion
        zip_code: identity_response_body.dig("address", "postal_code")
```
I think dig is safer for response objects.